js/lex.go

```diff
@@ -289,35 +288,35 @@ func (l *Lexer) consumeUnicodeEscape() bool {
 	return true
 }
 
-////////////////////////////////////////////////////////////////
-
-func (l *Lexer) consumeSingleLineToken(skip int) {
-	l.r.Move(skip)
```

Comment:
Because at the call site the `skip` integer its purpose is unclear. Besides, no other consume function uses this pattern so it is not consistent. Every consume function assumes it is at the beginning of what must be consumed, passing `skip` violates that.